packages/web/src/app/shared/doc-pages.ts

```diff
@@ -64,6 +64,16 @@ export const componentsDocPages: DocPage[] = [
     searchTerms: ['notifications', 'feedback', 'banner', 'flag', 'snackbar', 'message', 'confirmation'],
     relatedPages: ['toast', 'badge', 'cookies', 'validations'],
   },
+  {
+    title: 'App Bridge',
+    description:
+      'The app bridge can be used to open the selected meteringpoint in a different Elvia application. The list of application-links can be modified based on the systems needs. ',
+    docUrl: Pages.AppBridge,
+    absolutePath: '/components/' + Pages.AppBridge,
+    figmaUrl: 'https://www.figma.com/design/g0Dva16HcsfI0JUmSJY2lE/App-bridge?node-id=1-539&m=dev',
+    type: 'Component',
+    searchTerms: ['internal', 'application'],
```

Comment:
```suggestion
    searchTerms: ['internal', 'application', 'link', 'open', 'tab'],
```

Unsure about "tab". I was thinking about "open in new tab", but I can see how it can be confusing when we have a Tab UI element also

Reply:
I'm not too sure about the 'tab' myself, but the others are good!